ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1609,6 +1610,13 @@ ip_igmp_snooping:
 router_multicast:
   ipv4:
     routing: < true | false >
+    multipath: < none | deterministic >
+    multipath_deterministic_pick:  < color | router-id  >
```

Comment:
I don't think we should keep the keyword "pick", since this is not part of the EOS CLI. I believe the variable should be ``multipath_deterministic``. That should also be easily expandable to ``multipath_deterministic_colour`` if needed or in the future.

Reply:
had a chat with @ClausHolbechArista and making it a simpler solution and having only one option example

```yaml
    multipath: < none | deterministic | "deterministic color" | "deterministic router-id"  >
```